src/Api/Orbit.cpp

```diff
@@ -35,13 +35,17 @@ void EnqueueApiEvent(Types... args) {
 extern "C" {
 
 void orbit_api_start(const char* name, orbit_api_color color) {
-  EnqueueApiEvent<orbit_api::ApiScopeStart>(name, color);
+  auto return_address =
+      absl::bit_cast<uint64_t>(__builtin_extract_return_addr(__builtin_return_address(0)));
```

Comment:
Add a comment about the meaning of "0" in the call.

Reply:
Done.